Vostok.Metrics.System/Host/HostMonitorExtensions.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using JetBrains.Annotations;
+using Vostok.Logging.Abstractions;
+using Vostok.Metrics.System.Helpers;
+
+namespace Vostok.Metrics.System.Host
+{
+    [PublicAPI]
+    public static class HostMonitorExtensions
+    {
+        private static Action<ILog, TimeSpan, HostMetrics> loggingRule = (log, period, metrics) =>
+        {
+            // TODO: Add drive info and tcp state logging.
```

Comment:
I suggest omitting detailed TCP state staticstics here. Log messages aren't supposed to convey 100% of the information, just to give a short and useful summary.